Regressions/Tests/network_electric/electric_btms_test.py

```diff
@@ -0,0 +1,207 @@
+# Regression/scripts/Tests/network_all_electric/electric_btms_test.py
+# 
+# 
+# Created:  Sep 2024, S. Shekar
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  IMPORT
+# ----------------------------------------------------------------------------------------------------------------------
+# RCAIDE imports 
+import RCAIDE
+from RCAIDE.Framework.Core import Units  
+from RCAIDE.Library.Plots  import *
+from RCAIDE.Library.Methods.Performance.estimate_stall_speed        import estimate_stall_speed 
+
+# python imports     
+import sys
+import matplotlib.pyplot as  plt
+
+
+# local imports 
+sys.path.append('../../Vehicles')
+from Electric_Twin_Otter    import vehicle_setup, configs_setup 
+
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  REGRESSION
+# ----------------------------------------------------------------------------------------------------------------------  
+def main():           
+         
```

Comment:
you need to have truth values SOC, CL, anything ,and store them, then check them at the end of each simulation. See the other regressions for examples of how to do that

Reply:
Done